packages/wrapped-keys/src/lib/lit-actions-client/utils.ts

```diff
@@ -49,21 +47,32 @@ export function postLitActionValidation(
 }
 
 export function getLitActionCid(network: Network, actionType: LitActionType) {
+  // We already have guarantees that `actionType` is valid; it is not user provided and type-safe
+  assertNetworkIsValid(network);
+
   return LIT_ACTION_CID_REPOSITORY[actionType][network];
 }
 
 export function getLitActionCode(
   network: Network,
   actionType: LitActionType
 ): string {
-  const litActionCode = LIT_ACTION_CODE_REPOSITORY[actionType][network];
+  // We already have guarantees that `actionType` is valid; it is not user provided and type-safe
+  assertNetworkIsValid(network);
+
+  // No fuzzy validation needed here, because `setLitActionsCode()` validates its input
```

Comment:
Per my above comment, we want hard validation on `setLitActionsCode()`

Reply:
Done